.openpublishing.redirection.json

```diff
@@ -643,7 +643,7 @@
             "redirect_url": "https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki"
         },
         {
-            "source_path": "docs/framework/standard/design-guidelines/dispose-pattern.md",
+            "source_path": "docs/standard/design-guidelines/dispose-pattern.md",
             "redirect_url": "~/dotnet/standard/garbage-collection/implementing-dispose"
```

Comment:
@rpetrusha I've never seen the `redirect_url` starting with `~`. Is it equivalent to
```json
"redirect_url": "/dotnet/standard/garbage-collection/implementing-dispose"
```
?

Reply:
You're right @pkulikov. It shouldn't.